src/Entity/Hearing.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+namespace App\Entity;
+
+use App\Logging\LoggableEntityInterface;
+use App\Repository\HearingRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Timestampable\Traits\TimestampableEntity;
+use Symfony\Component\Uid\Uuid;
+
+/**
+ * @ORM\Entity(repositoryClass=HearingRepository::class)
+ */
+class Hearing implements LoggableEntityInterface
+{
+    use TimestampableEntity;
+
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="uuid", unique=true)
+     */
+    private $id;
+
+    /**
+     * @ORM\OneToMany(targetEntity=HearingPost::class, mappedBy="hearing")
+     */
+    private $hearingPosts;
+
+    /**
+     * @ORM\OneToOne(targetEntity=CaseEntity::class, mappedBy="hearing", cascade={"persist", "remove"})
+     */
+    private $caseEntity;
+
+    /**
+     * @ORM\Column(type="boolean", options={"default":"0"})
+     */
+    private $complainantHasNoMoreToAdd = false;
+
+    /**
+     * @ORM\Column(type="boolean", options={"default":"0"})
+     */
+    private $counterpartHasNoMoreToAdd = false;
+
+    /**
+     * @ORM\Column(type="boolean", options={"default":"0"})
+     */
+    private $hasNewHearingPost = false;
+
+    /**
+     * @ORM\Column(type="boolean", options={"default":"0"})
+     */
+    private $hasBeenStarted = false;
+
+    /**
+     * @ORM\Column(type="date", nullable=true)
+     */
+    private $startDate;
+
+    /**
+     * @ORM\Column(type="boolean", options={"default":"0"})
+     */
+    private $hasFinished = false;
```

Comment:
Consider using a `?DateTimeInterface` value here.

Reply:
Fixed in 3d4a02477b062529bd25cb3e4c1e4fbc8041c8c9.